appendices/special-characters/variable-index.md

```diff
@@ -1,40 +1,39 @@
 # Variable Index
 
-## **Syntax**
+## Syntax
 
-```text
+```G1ANT
 ♥variable⟦position⟧
```

Comment:
this syntax is very confusing :p you can leave it as you like but please consider removing syntax from this topic:)

Reply:
what do you mean? What's confusing exactly?

Reply:
well... it's nothing really. It's just that I don't really get why we have syntax for such small elements in G1ANT like variable indexes. And when I started reading it, I didn't know what was going on there :p until I read the description part of course. And position is not the only value for an index, it can also be for example: ♥error⟦code⟧
but you can leave it as it is, I left a comment here because for me the syntax part was more confusing than helpful :D